lighthouse-core/gather/gatherers/anchor-elements.js

```diff
@@ -5,11 +5,9 @@
  */
 'use strict';
 
-/* global getNodeDetails */
-
 const FRGatherer = require('../../fraggle-rock/gather/base-gatherer.js');
 const dom = require('../driver/dom.js');
-const pageFunctions = require('../../lib/page-functions.js');
+const {getElementsInDocument, getNodeDetails} = require('../../lib/page-functions.js');
```

Comment:
(you probably already know this, just wanted to make it explicit)

FWIW this is what breaks terser. this is basically 

```
const getElementsInDocument = require('../../lib/page-functions.js').getElementsInDocument;
```

which allows terser to do w/e it wants with the name here since it is just local.